src/logger-mixin/mixins/README.md

```diff
@@ -0,0 +1,99 @@
+# Mixins
+
+This directory contains mixins for `Application`.
+
+## Overview
+
+A mixin is a class which take the input of a base class and adds / modifies existing functions / status values and returns a new class which can be instantiated. The type of the returned class will still be the base class. Mixins let you extend the BaseClass by adding more functions to it, overriding existing ones or modifiying their behavior.
+
+## Basic Usage
+
+The general idea behind using a mixin is to pass in the BaseClass to the Mixin and create a new class. 
+
+**Example**
+```
+cosnt newClass = MixinClass(BaseClass);
+```
+
+Mixins can be nested, as such you can do the following:
+```
+cosnt newClass = MixinClass1(MixinClass2(BaseClass))
+```
+
+### LoggerMixin
+
+LoggerMixin adds capabilities to a LoopBack Next Application by adding a `.logger()` function that allows used to bind a Logger class to `Context` automatically. The binding key will be `loggers.${Class.name}` where `Class.name` is the name of the Logger class being bound. The Mixin also overrides existing `.component()` function so that components are also capable of providing Logger's to be bound automatically. 
```

Comment:
`...function that allows used to...` --> `that is used to` or `that allows a`

Reply:
`...also capable of providing Logger's to be...` --> `also capable of providing their own Logger implementations automatically`

Reply:
Also, split this line into multiple lines, it's far too long.